### ISSUE TITLE
[low] Color Contrast: Many Pages: Green icon over white background fails contrast ratio

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5511]
## Action Performed:
Prerequisite: the user is logged in.
1. Using Chrome, open the site https://staging.new.expensify.com/settings
2. Once the Settings page opens, measure the color contrast ratio of the green 'online' icon against white background.

## Expected Result:
The visual presentation of meaningful graphical objects or interactive components should have a contrast ratio of at least 3.00:1 with the surrounding background color.

## Actual Result:
The green icon (#03D47C) against white background (#FFFFFF) has a color contrast ratio of 1.96:1.

HTML:
`<div class="css-1dbjc4n r-u8s1d" style="align-items: center; background-color: rgb(3, 212, 124); border-color: rgb(255, 255, 255); border-radius: 8px; border-width: 2px; bottom: 4px; height: 16px; justify-content: center; right: 4px; transform: rotate(0deg) scale(1); width: 16px; z-index: 10; -webkit-box-align: center; -webkit-box-pack: center;"></div>`

CSS:
`{align-items: center;
background-color: rgb(3, 212, 124);
border-color: rgb(255, 255, 255);
border-radius: 8px;
border-width: 2px;
bottom: 4px;
height: 16px;
justify-content: center;
right: 4px;
transform: rotate(
0deg
) scale(1);
width: 16px;
z-index: 10;
-webkit-box-align: center;
-webkit-box-pack: center;
}`

Other occurrences:
1. Chat ->The color contrast ratio of green 'online' icon (#03D47C) against white background (#FFFFFF) is 1.96:1.

## Workaround:
No workaround

## Area issue was found in:
Settings overlay, Chat page

## Failed WCAG checkpoints
1.4.11

## User impact:
Some users won't be able to see the icons.

## Suggested resolution:
Use a darker green color to achieve a color contrast ratio of at least 3.00:1. For example, `#03AA65`, which will give the contrast ratio of 3.02:1 against white background.

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**
![Bug5182613_Online_green_icon_fails_color_contrast_ratio](https://user-images.githubusercontent.com/88733897/134730867-1566f482-1453-4a1a-b701-b97350c9af09.png)
![Bug5182613_Chat_page_-_green_online_status_indicator_on_white_background_fails_contrast_requirements](https://user-images.githubusercontent.com/88733897/134730875-69b07f42-db5e-426b-9a37-bf6b6697a21b.png)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
